docs/speaker-diversity-policy.md

```diff
@@ -0,0 +1,18 @@
+# Speaker diversity policy
+
+Due to the fact that we want our events to have a more diverse and inclusive range of speakers at events we recommend that you implement a speaker diversity policy and stick to it for your session selections. It is _optional_ but it will be appreciated in the community.
```

Comment:
```suggestion
Because we want our events to have a more diverse and inclusive range of speakers at events, we recommend that you implement a speaker diversity policy and stick to it for your session selections. It is _optional_ but it will be appreciated by the community.
```